js/bitstamp.js

```diff
@@ -526,6 +612,16 @@ module.exports = class bitstamp extends Exchange {
         };
     }
 
+    parseTradingFee (balances, symbol) {
+        const market = this.market (symbol);
+        const tradeFee = this.safeFloat (balances, market.id + '_fee');
```

Comment:
We will need to use `market['id']` instead of `market.id` here and everywhere throughout the rest of the file, otherwise it is violation of this guideline: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys.

Reply:
Fixed